client/trino-cli/src/main/java/io/trino/cli/Trino.java

```diff
@@ -53,7 +53,10 @@ public static CommandLine createCommandLine(Object command)
                 .registerConverter(HostAndPort.class, HostAndPort::fromString)
                 .registerConverter(Duration.class, Duration::valueOf);
 
-        getConfigFile().ifPresent(file -> commandLine.setDefaultValueProvider(new CommandLine.PropertiesDefaultProvider(file)));
+        getConfigFile().ifPresent(file -> {
+            System.err.println("Loading default configuration from " + file.getPath());
```

Comment:
Is this is a common way to display warnings in CLI?

Reply:
No, there isn't. We are printing to stderr (see Console for example)